lib/token-request.js

```diff
@@ -321,7 +321,7 @@ TokenRequest.prototype._getTokenUsernamePasswordFederated = function(username, p
       return;
     }
 
-    this._performUsernamePasswordForAccessTokenExchange(this._userRealm.federationActiveAuthUrl, username, password, callback);
+    this._performUsernamePasswordForAccessTokenExchange(this._userRealm.federationActiveAuthUrl, constants.WSTrustVersion.UNDEFINED, username, password, callback);
```

Comment:
As I understand it, the federatedActiveAuthUrl is a substitute for making the Mex request. If you make the Mex request, you get back a list of end points and protocols that you can use to authenticate. The federatedActiveAuthUrl is one of those endpoints, but you only get 1 instead of the 12 options you can get in the Mex response.

The call to _performUsernamePasswordForAccessTokenExchange makes a WSTrust request, so we need to know what version to use for that request.